plotheatmap/plotheatmap.xml

```diff
@@ -0,0 +1,103 @@
+    </command>
+    <inputs>
+        <section name="main" title="Main parameters" expanded="True">
+            <param name="count_matrix" type="data" format="data" label="Output dataset 'count_matrix' from tool countMatrixForEdger"/>
+            <param name="stats_matrix" type="data" format="data" label="Output dataset 'output' from tool edgeR"/>
+        </section>
+        <section name="filter" title="Filtering parameters" expanded="True">
+            <param name="log_fc" type="float" value="0.5" label="A log FC threshold"/>
+            <param name="log_cpm" type="float" value="0.5" label="A log CPM threshold"/>
+            <param name="pvalue" type="float" value="0.5" label="A PValue threshold">
+                <validator type="in_range" min="0" max="1" />
+            </param>
+            <param name="fdr" type="float" value="0.5" label="A FDR threshold">
+                <validator type="in_range" min="0" max="1" />            
+            </param>
+        </section>
+        <section name="heatmap" title="Heatmap parameters">
+            <param name="cluster_row" type="select" label="Determines if the row dendrogram should be reordered">
+                <option value="No" >No </option>
+                <option value="Yes" selected="true" >Yes </option>
+            </param>
+            <param name="cluster_col" type="select" label="Determines if the column dendrogram should be reordered">
+                <option value="No" selected="true">No </option>
+                <option value="Yes" >Yes </option>
+            </param>
+
+            <param name="hclust_method" type="select" label="Hierarchical clustering method">
+                <option value="ward.D" >ward.D </option>
+                <option value="ward.D2" >ward.D2 </option>
+                <option value="single" >single </option>
+                <option value="complete" >complete </option>
+                <option value="average" selected="true">average </option>
+                <option value="mcquitty" >mcquitty </option>
+                <option value="median" >median </option>
+                <option value="centroid" >centroid </option>
+            </param>
+        </section>
+
+
+       <section name="plotparm" title="Plot parameters">
+            <param name="col_marg" type="float" value="7" label="Margins for column names"/>
+            <param name="row_marg" type="float" value="8" label="Margins for row names"/>
+            <param name="pdf_width" type="float" value="7" label="Width of the graphics region in inches (pdf width)" />
+            <param name="pdf_height" type="float" value="7" label="Height of the graphics region in inches (pdf height)" />
+        </section>
+
+    </inputs>
+    <outputs>
+       <data format="pdf" name="haetamappdf" from_work_dir="heatmap.pdf" >
+            <filter>(outftype == 'pdf')</filter>
+        </data>
+        <data name="rscript_log" format="data" />
+        <data name="rerror_log" format="data" />
```

Comment:
I think you can remove the `rscript_log` and `rerror_log` outputs, they seems to unnecessarily clutter the history. These information will be available anyway when clicking the "i" icon of the `haetamappdf` output dataset.